src/file-uploader/file-uploader.component.ts

```diff
@@ -31,11 +31,11 @@ const noop = () => { };
 					(keyup.enter)="fileInput.click()"
 					(keyup.space)="fileInput.click()"
 					[ngClass]="{'bx--file-browse-btn--disabled': disabled}"
+					role="button"
```

Comment:
Label element cannot have the ARIA role 'button'.

A possible solution to this would be to move the tabindex to the child div element. We have the `for` property assigned to the div, so I think that would give screen readers context & it has the `role` set to `button` which makes sense to allow it to be focusable 🤔

Reply:
Sure, I can try that.
On the same note about that `<label>` - there's 2 label elements in the same component which might also be a part of the problem

Reply:
Hmm, you're right, there should only be one.  I'll need to play around with it to figure out the best way to approach this. Removing one of the labels will still show the `a11y role 'none'` error unfortunately.

Reply:
I ended up removing that whole secondary label element as it's not really needed

Reply:
> Hmm, you're right, there should only be one. I'll need to play around with it to figure out the best way to approach this. Removing one of the labels will still show the `a11y role 'none'` error unfortunately.

Ah, you're right

Reply:
Wait, I was looking at the wrong branch. Here's what the checker is showing now - there's no 'none' complaints.

![image](https://github.com/carbon-design-system/carbon-components-angular/assets/1253469/9ddd87c1-7b35-4f55-8de3-5c550cb2ea3c)

Reply:
@Akshat55 I made the change you've requested and the PR is ready for another review